### PR TITLE
[ENH]: Fix copy-paste errors and add Ar to msis.py

### PR DESCRIPTION
**Pull request by FGuenzkofer**
- Fixed H, He and N masses
- Edited He constant according to added [2]
- Added Ar according to [2]

### CODE REVIEW

File: flipchem/msis.py [hunk 1]
```diff
@@ -59,11 +62,12 @@ def compute_ion_neutral_collfreq(densities, Tn, mi, Ti=None):
     #    n_in = const * n_n / sqrt(m_i*(m_i+m_n))
     # for each ion. Here's the constants for H+, He+, N+, O+, N2+, and O2+:
     Hconst  = 2.118436361550408e-15
-    Heconst = 2.372016271579909e-15 # assuming 100% He-4
+    Heconst = 4.796838917e-15 #old value: 2.372016271579909e-15 # assuming 100% He-4
```
Comment (asreimer) [inline flipchem/msis.py:65]:
```suggestion
    Heconst = 4.796838917e-15 # From 10.1103/PhysRevLett.120.123203
```

File: flipchem/msis.py [hunk 2]
```diff
@@ -59,11 +62,12 @@ def compute_ion_neutral_collfreq(densities, Tn, mi, Ti=None):
...
     Nconst  = 1.0293931179488746e-14
     Oconst  = 9.0841307137989e-15
     N2const = 1.8168261427597804e-14
     O2const = 1.8518806819759527e-14
+    Arconst = 4.291797148e-14
```
Comment (asreimer) [inline flipchem/msis.py:70]:
```suggestion
    Arconst = 4.291797148e-14 # From 10.1103/PhysRevLett.120.123203
```

File: flipchem/msis.py (hunk 1 above)
Comment (asreimer) [reply flipchem/msis.py:65]: We'll have the old value in the git history, so we can remove that, but it's probably useful to start leaving comments about where these constants are coming from now.

**Review by asreimer — COMMENTED** [review]
Thanks! This looks great. I swapped over to the `develop` branch, where I like to work on features and non-critical bug fixes before doing a release.

Also made a few suggestions for comments. Let me know what you think.

**Comment by FGuenzkofer** [comment]
Good idea to clarify constant sources in comments. Also edited one comment line to make it more comprehensible.

**Comment by asreimer** [comment]
This looks great! Thanks again for making these changes.

**Pull request merged**